tests/e2e/e2e_suite_test.go

```diff
@@ -49,7 +49,8 @@ var (
 	knownFlake          bool
 	accumulatedTestLogs []string
 
-	kvmEmulation bool
+	kvmEmulation         bool
+	unsupportedOverrides map[string]string
```

Comment:
unused, remove (a LINTER could have catch this...)

Reply:
sure, but that is also useless imho as none of the vars are truly aligned. 
It's up for debate I guess, but I don't find that meaningful, other than I actually didn't change that code, so I should not have let my editor realign it.

Reply:
thanks for clarifying this bit, I misread your comment. appologies.